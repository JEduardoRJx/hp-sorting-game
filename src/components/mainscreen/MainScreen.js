import React from 'react';
import './MainScreen.scss'

const MainScreen = () => {
  return(
    <section className='main-screen'>
      <div className='icon-wrapper sorting-hat'>
        <img className='main-screen-img' 
          src={require('../../images/sortingHat.svg')} 
          alt='sorting hat' />
          <label>Discover Your House</label>
      </div>
      <div className='icon-wrapper golden-snitch'>
        <img className='main-screen-img' 
          src={require('../../images/snitch.svg')} 
          alt='golden snitch' />
          <label>Play Game</label>

      </div>
      <div className='icon-wrapper instructions'>
        <img className='main-screen-img' 
          src={require('../../images/instructions.svg')} 
          alt='instructions' />
          <label>Instructions</label>
      </div>
      <div className='icon-wrapper characters'>
        <img className='main-screen-img' 
          src={require('../../images/characters.svg')} 
          alt='characters' />
          <label>View All Characters</label>
      </div>
      <div className='icon-wrapper wand'>
        <img className='main-screen-img' 
          src={require('../../images/wand.svg')} 
          alt='wand view spells' />
          <label>View All Spells</label>
      </div>
    </section>
  )
}

export default MainScreen;