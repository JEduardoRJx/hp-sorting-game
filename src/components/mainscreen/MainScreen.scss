.main-screen {
  background-color: blue;
  height: 90vh;
  display: grid;
  grid-template-columns: 1fr 1fr 1fr 1fr;
  grid-template-rows: 1fr 1fr 1fr;
  align-items: center;
  justify-items: center;
  // row-gap: 10px;
}

.icon-wrapper {
  background-color: red;
  display: flex;
  flex-direction: column;
  align-items: center;
  
}

.main-screen-img {
  display: flex;
  height: 185px;
  &:hover {
    transform: scale(1.1);
    cursor: pointer;
  }
}

.sorting-hat {
  grid-column: 1/5;
}

.golden-snitch,
.characters {
  grid-column: 1/3;
}

.instructions,
.wand {
  grid-column: 3/5;
}

label {
  font-size: 1.5em;
  margin-top: 10px;
}