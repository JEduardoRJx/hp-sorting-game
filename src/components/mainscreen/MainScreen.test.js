import React from 'react';
import { shallow } from 'enzyme';
import MainScreen from './MainScreen'

describe('MainScreen', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<MainScreen />)
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})