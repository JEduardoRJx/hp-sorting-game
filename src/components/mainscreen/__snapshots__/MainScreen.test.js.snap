// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainScreen should match snapshot 1`] = `
<section
  className="main-screen"
>
  <Connect(PlayerInfo) />
  <Connect(Navigations) />
</section>
`;
